docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: "3"
+
+services:
+  dev-node:
+    build: .
+    command: ./target/release/mashnet-node --dev --ws-port 9944 --ws-external
+    ports:
+      - 9944:9944
+  sdk:
+    build:
+      context: https://github.com/KILTprotocol/sdk-js.git#rf-docker-compose
```

```suggestion
      context: https://github.com/KILTprotocol/sdk-js.git
```
will have to change once the Dockerfile is merged to develop on the sdk